cucumber/api/features/step_definitions/user_steps.rb

```diff
@@ -33,8 +33,12 @@
   Role.create(role_id: roleid)
 end
 
+def host?(login)
```

Comment:
is it some ruby standart for the name ?
Please add comment to explain the method and make it private if possible in ruby

Reply:
Yes. It's a ruby standard for predicate functions.
Comment added